code-challenges.js

```diff
@@ -30,40 +43,97 @@ const temperature2 = 350
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPointTemp(temperature1))
+console.log(boilingPointTemp(temperature2))
+console.log(boilingPointTemp(temperature3))
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// combine 2 arrays......concat? also well need to return the combined length
+// varible combining arrays and then another varible strictly for combining the arrays
+// return the the combined arrays 
+// create another varible that I can console.log the the combined arrays
```

Comment:
The pseudo code is unclear here. You said variable combining the arrays, then another variable strictly for combining the arrays. What does that mean? Make sure your pseudo code is clear